main.go

```diff
@@ -144,34 +144,36 @@ func (app *App) MakeServer() *echo.Echo {
 	}
 
 	// Front
-	t := NewTemplate(app)
-	e.Renderer = t
-	e.GET("/", FrontRoot(app))
-	e.GET("/web/admin", FrontAdmin(app))
-	e.GET("/web/create-player-challenge", FrontCreatePlayerChallenge(app))
-	e.GET("/web/manifest.webmanifest", FrontWebManifest(app))
-	e.GET("/web/player/:uuid", FrontPlayer(app))
-	e.GET("/web/register-challenge", FrontRegisterChallenge(app))
-	e.GET("/web/registration", FrontRegistration(app))
-	frontUser := FrontUser(app)
-	e.GET("/web/user", frontUser)
-	e.GET("/web/user/:uuid", frontUser)
-	e.POST("/web/admin/delete-invite", FrontDeleteInvite(app))
-	e.POST("/web/admin/new-invite", FrontNewInvite(app))
-	e.POST("/web/admin/update-users", FrontUpdateUsers(app))
-	e.POST("/web/create-player", FrontCreatePlayer(app))
-	e.POST("/web/delete-player", FrontDeletePlayer(app))
-	e.POST("/web/delete-user", FrontDeleteUser(app))
-	e.POST("/web/login", FrontLogin(app))
-	e.POST("/web/logout", FrontLogout(app))
-	e.POST("/web/register", FrontRegister(app))
-	e.POST("/web/update-player", FrontUpdatePlayer(app))
-	e.POST("/web/update-user", FrontUpdateUser(app))
-	e.Static("/web/public", path.Join(app.Config.DataDirectory, "public"))
-	e.Static("/web/texture/cape", path.Join(app.Config.StateDirectory, "cape"))
-	e.Static("/web/texture/default-cape", path.Join(app.Config.StateDirectory, "default-cape"))
-	e.Static("/web/texture/default-skin", path.Join(app.Config.StateDirectory, "default-skin"))
-	e.Static("/web/texture/skin", path.Join(app.Config.StateDirectory, "skin"))
+	if app.Config.EnableFrontend {
+		t := NewTemplate(app)
+		e.Renderer = t
+		e.GET("/", FrontRoot(app))
+		e.GET("/web/admin", FrontAdmin(app))
+		e.GET("/web/create-player-challenge", FrontCreatePlayerChallenge(app))
+		e.GET("/web/manifest.webmanifest", FrontWebManifest(app))
+		e.GET("/web/player/:uuid", FrontPlayer(app))
+		e.GET("/web/register-challenge", FrontRegisterChallenge(app))
+		e.GET("/web/registration", FrontRegistration(app))
+		frontUser := FrontUser(app)
+		e.GET("/web/user", frontUser)
+		e.GET("/web/user/:uuid", frontUser)
+		e.POST("/web/admin/delete-invite", FrontDeleteInvite(app))
+		e.POST("/web/admin/new-invite", FrontNewInvite(app))
+		e.POST("/web/admin/update-users", FrontUpdateUsers(app))
+		e.POST("/web/create-player", FrontCreatePlayer(app))
+		e.POST("/web/delete-player", FrontDeletePlayer(app))
+		e.POST("/web/delete-user", FrontDeleteUser(app))
+		e.POST("/web/login", FrontLogin(app))
+		e.POST("/web/logout", FrontLogout(app))
+		e.POST("/web/register", FrontRegister(app))
+		e.POST("/web/update-player", FrontUpdatePlayer(app))
+		e.POST("/web/update-user", FrontUpdateUser(app))
+		e.Static("/web/public", path.Join(app.Config.DataDirectory, "public"))
+		e.Static("/web/texture/cape", path.Join(app.Config.StateDirectory, "cape"))
```

Comment:
These `texture` routes are used for skins/capes and should not be disabled. They should probably be renamed to `/drasl/texture/...` though (like the `/drasl/api` routes) since they're not exclusively part of the web interface.

Reply:
partially resolved in d856a85a

I don't know what else should be done for renaming texture routes. So I won't touch anything to not break something.

Reply:
No problem, I'll do it in a follow-up